programs/static-files-disk-uploader/static-files-disk-uploader.cpp

```diff
@@ -66,8 +66,14 @@ static void processFile(const fs::path & file_path, const fs::path & dst_path, b
         std::shared_ptr<WriteBuffer> dst_buf;
 
         /// test mode for integration tests.
+        // todo should this bypass the proxy?
```

Comment:
from original logic it should

Reply:
This code belongs to what looks like a standalone tool: https://clickhouse.com/docs/operations/utilities/static-files-disk-uploader

I think it makes no sense to read proxy from clickhouse xml configuration files, BUT it might make sense to read from the environment variables.

Suppose someone is running this tool in a network that has a proxy server, it won't work if it bypasses the proxy. (unless the target is a server within the same network)